website/content/docs/k8s/installation/deployment-configurations/clients-outside-kubernetes.mdx

```diff
@@ -35,14 +48,69 @@ annotation `consul.hashicorp.com/auto-join-port` to an integer value or
 a named port to specify the port for the auto-join to return. This enables
 different pods to have different exposed ports.
 
-## Networking
+### Auto-join on the Pod network
+In the default Consul Helm chart installation, the Consul clients and servers
+are deployed as pods and routeable only via their `podIP:containerPort` for
+gossip/RPC. This would mean any external client agents joining would need to
+be able to have connectivity to those pod IPs.
```

Comment:
```suggestion
In the default Consul Helm chart installation, Consul clients and servers
are routable only via their pod IPs for
gossip and server RPC (technically, HTTP API calls to Consul clients can also be made through host IPs). This means any external client agents joining the Consul cluster running on Kubernetes would need to
be able to have connectivity to those pod IPs.

In many hosted Kubernetes environments, you will need to explicitly configure your hosting provider to ensure that pod IPs are routable from external VMs. See [Azure AKS CNI](https://docs.microsoft.com/en-us/azure/aks/concepts-network#azure-cni-advanced-networking), [AWS EKS CNI](https://docs.aws.amazon.com/eks/latest/userguide/pod-networking.html) and [GKE VPC-native clusters](https://cloud.google.com/kubernetes-engine/docs/concepts/alias-ips).
```

* I think saying "in the default install consul clients and servers are deployed as pods..." might be a bit confusing because it sounds like there's another install where they're not deployed as pods.
* added a bit more colour but feel free to edit

Reply:
Love it!